pandas/tests/arithmetic/test_object.py

```diff
@@ -323,3 +324,51 @@ def test_rsub_object(self):
 
         with pytest.raises(TypeError):
             np.array([True, pd.Timestamp.now()]) - index
+
+    @pytest.mark.parametrize(
```

Comment:
I think you should move this to:

tests/indexes/test_subclass.py

Reply:
I don't see an indexes/test_subclass.py. Start a new file for this?